applications/nrf_desktop/description.rst

```diff
@@ -0,0 +1,1082 @@
+
+   * NCS keyboard - The Fast Pair Provider meant to be used with keyboards:
+
+      * Device Name: NCS keyboard
+      * Model ID: ``0x52FF02``
+      * Anti-Spoofing Private Key (base64, uncompressed): ``8E8ulwhSIp/skZeg27xmWv2SxRxTOagypHrf2OdrhGY=``
+      * Device Type: Input Device
+      * Notification Type: Fast Pair
+      * Data-Only connection: true
+      * No Personalized Name: false
+
+   * NCS gaming mouse - Fast Pair Provider meant to be used with gaming mice:
+
+      * Device Name: NCS gaming mouse
+      * Model ID: ``0x8E717D``
+      * Anti-Spoofing Private Key (base64, uncompressed): ``dZxFzP7X9CcfLPC0apyRkmgsh3n2EbWo9NFNXfVuxAM=``
+      * Device Type: Input Device
+      * Notification Type: Fast Pair
+      * Data-Only connection: true
+      * No Personalized Name: false
+
+   See :ref:`ug_bt_fast_pair_provisioning` documentation for the following information:
+
+   * Registering a Fast Pair Provider
+   * Provisioning a Fast Pair Provider in |NCS|
+
+.. nrf_desktop_fastpair_important_start
+
+.. important::
+   This is the debug Fast Pair provisioning data obtained by Nordic for the development purposes.
+    Do not use it in production.
+
+   To test with the debug mode Model ID, you must configure the Android device to include the debug results while displaying the nearby Fast Pair Providers.
+   For details, see `Verifying Fast Pair`_ in the GFPS documentation.
+
+.. nrf_desktop_fastpair_important_end
+
+.. _nrf_desktop_testing_steps:
+
+Testing
+=======
+
+You can build and test the application in various configurations.
+The following procedure refers to the scenario where the gaming mouse (nRF52840 Gaming Mouse) and the keyboard (nRF52832 Desktop Keyboard) are connected simultaneously to the dongle (nRF52840 USB Dongle).
+
+After building the application with or without :ref:`specifying the build type <nrf_desktop_selecting_build_types>`, test the nRF Desktop application by performing the following steps:
+
+1. Program the required firmware to each device.
+#. Insert the :ref:`debug board or bypass board <nrf_desktop_debugging>` into the mouse to make sure it is powered.
+#. Turn on both mouse and keyboard.
+   **LED1** on the keyboard and **LED1** on the mouse start breathing.
+#. Plug the dongle to the USB port.
+   The blue **LED2** on the dongle starts breathing.
+   This indicates that the dongle is scanning for peripherals.
+#. Wait for the establishment of the Bluetooth connection, which happens automatically.
+   After the Bluetooth connection is established, the LEDs stop breathing and remain turned on.
+   You can now use the devices simultaneously.
+
+   .. note::
+        You can manually start the scanning for new peripheral devices by pressing the **SW1** button on the dongle for a short time.
+        This might be needed if the dongle does not connect with all the peripherals before scanning is interrupted by a timeout.
```

Comment:
```suggestion
   .. note::
      You can manually start the scanning for new peripheral devices by pressing the **SW1** button on the dongle for a short time.
      This might be needed if the dongle does not connect with all the peripherals before scanning is interrupted by a timeout.
```